app/views/policies/_list_body.html.slim

```diff
@@ -1,46 +1,14 @@
-tbody
-  tr
-    td
-      | policy jivan vima nigam
-    td
-      | Lic 
-    td
-      | Lic insurance companey pvt.ltd
-    td
-      | karan bamaniya
-    td
-      | 2/2/2006
-    td 
-      |2/2/2021
-    td
-      |action
-  tr
-    td
-      | policy jivan vima nigam
-    td
-      | Lic 
-    td
-      | Lic insurance companey pvt.ltd
-    td
-      | karan bamaniya
-    td
-      | 2/2/2006
-    td 
-      |2/2/2021
-    td
-      |action
-  tr
-    td
-      | policy jivan vima nigam
-    td
-      | Lic 
-    td
-      | Lic insurance companey pvt.ltd
-    td
-      | karan bamaniya
-    td
-      | 2/2/2006
-    td 
-      |2/2/2021
-    td
-      |action
\ No newline at end of file
+- policies.each do | policy |
+  tbody
+    tr
+      td
+        = policy.personal_info.first_name
```

Comment:
@GauravMakwana-BTC : Display full name instead of first name, create a function on the model which will return the full name when you write like policies.personal_info.full_name